boa/contracts/vyper/vyper_contract.py

```diff
@@ -85,10 +176,27 @@ def __init__(self, compiler_data, filename=None):
     def __call__(self, *args, **kwargs):
         return self.deploy(*args, **kwargs)
 
-    def deploy(self, *args, **kwargs):
+    def deploy(self, *args, verify: bool = False, explorer: Optional[str] = None, **kwargs):
```

Comment:
We could coalesce both parameters into a single one - no need for a boolean - maybe even a list of explorers. Besides being more succinct and flexible, it allows us to delete the validation below.